words/utils.py

```diff
@@ -52,7 +52,13 @@ def translate(self, words, source_language, target_language):
         words = list(set(words))
 
         return [
-            {"original": word, "translation": f"t_{word}", "sl": "en", "tl": "pl"} for word in words
+            {
+                "original": word,
+                "translation": f"t_{word}",
+                "sl": source_language,
```

Comment:
"sl" i "tl" to trochę mało znaczące nazwy

Reply:
tak jest w API google